src/types/page.urls.ts

```diff
@@ -0,0 +1,16 @@
+export enum urlParams {
+    PARAM_ZIP_NUMBER = "zipNumber",
+    PARAM_COMPANY_NUMBER = "companyNumber",
+    PARAM_TRANSACTION_ID = "transactionId",
+    PARAM_SUBMISSION_ID = "submissionId",
+}
+
+export enum URL_QUERY_PARAM {
```

Comment:
Every value in this enum is also in the above one, is there a reason we need both?